src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -147,6 +147,14 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 			cfg.Redpanda.SeedServers = []config.SeedServer{}
 			cfg.Redpanda.SeedServers = seeds
 
+			// The Redpanda binary defaults to the root-driven style of cluster
+			// formation. In bootstrapping a new cluster, use seeds-driven
+			// cluster formation.
+			if cfg.Redpanda.Other == nil {
+				cfg.Redpanda.Other = make(map[string]interface{})
+			}
+			cfg.Redpanda.Other["empty_seed_starts_cluster"] = false
```

Comment:
Should we verify that `seeds` has at least one item?

Also maybe print a warning if `seeds` size is less than 3?

Reply:
Done